1-CONTRIBUTION-GUIDE/README.md

```diff
@@ -106,6 +109,30 @@ To make sure your template is added to Azure.com index, please follow these guid
 1. Each PR will run through the [arm-ttk](https://github.com/Azure/arm-ttk) to ensure best practices
 1. Part of the pre-merge checks will be a deployment to both the Public and USGov clouds
 
+## Bicep support
+
+We encourage new samples to be written directly in [Bicep](https://docs.microsoft.com/en-us/azure/azure-resource-manager/bicep/overview) and encourage existing samples to be converted to support Bicep.
+
+1. The bicep file must be named **main.bicep**
+1. The **azuredeploy.json** file must still be included, and must be generated by the bicep compiler (use this command: `build main.bicep --outfile azuredeploy.json`)
+1. The [**README.md**](sample-README.md) file must include a link to the bicep badge
+1. The parameter file must still be named **azuredeploy.parameters.json**
+
+An easy way to convert an existing sample to support Bicep is to use the Bicep decompiler:
+
+```sh
+bicep decompile azuredeploy.json --outfile main.bicep
+```
+
+The decompiler is not guaranteed to produce correct Bicep code from JSON, so you will need to inspect and modify the **main.bicep** file afterwards (and then use `bicep build` to generate the matching **azuredeploy.json** file). Some general guidelines:
+
+1. Rename (F2 in VsCode) parameters and variables to be camel-cased.
+1. Rename resource symbolic names to a logical, short name, such as 'storage' or 'vmStorage' for a resource of type `Microsoft.Storage/storageAccounts`. Remove `Name` from the symbolic name if the decompiler creates it that way.
+1. Remove `_var`, `_param` and `_resource` prefixes if they are present in variables, parameters and resources.
+1. Use bicep concepts when possible
+1. See [decompiling](https://github.com/Azure/bicep/blob/main/docs/decompiling.md) for current limitations of the Bicep decompiler.
+1. See also [Best practices for Bicep](https://docs.microsoft.com/en-us/azure/azure-resource-manager/bicep/best-practices)
+
```

Comment:
we should add a section for "sorting" - as we have for the json...  I think we'll have to work through ordering of the decorators (that were previously properties) but if you want to take a stab at it, that's a fine place to start...

Reply:
Took a stab...